rspack.static-viz.config.js

```diff
@@ -115,6 +116,7 @@ module.exports = env => {
         cljs: devMode ? CLJS_SRC_PATH_DEV : CLJS_SRC_PATH,
         "metabase-lib": LIB_SRC_PATH,
         "metabase-types": TYPES_SRC_PATH,
+        embedding: EMBEDDING_SRC_PATH,
```

Comment:
I updated here as well since I have already updated `rspack.config.js` with the same change.